lua/telescope/make_entry.lua

```diff
@@ -615,8 +615,14 @@ end
 
 function make_entry.gen_from_apropos(opts)
   local sections = {}
-  for _, section in ipairs(opts.sections) do
-    sections[section] = true
+  if #opts.sections == 1 and opts.sections[1] == 'ALL' then
```

Comment:
Lets to `tolower`. and == 'all'

Reply:
I've chosen `'ALL'` because it has less occurrence to see than `'all'`. But if `'all'` is enough to see rarely, then I use `'all'` for this. How do you think?

Reply:
Yeah i see. Lets keep it that way :) I will merge